images/archlinux/Containerfile

```diff
@@ -0,0 +1,24 @@
+FROM docker.io/library/archlinux:base-devel
+
+ENV NAME=archlinux-toolbox VERSION=base-devel
```

Comment:
We should replace this `ENV` with `ARG`, like @travier [did](https://github.com/containers/toolbox/commit/9506173f88dc26bfb0229edea0d39699d636d9e2) for the Fedora images, to prevent `NAME` and `VERSION` from leaking into the interactive command line environment inside the container.

Secondly, I think the image should be called `arch-toolbox`, not `archlinux-toolbox`, to match the `ID` in `os-release(5)` on Arch Linux.  Toolbx uses `os-release(5)` in various different places and the concept of `ID` is embedded very deeply in the logic.  I think it will be simpler if we stuck to the `os-release(5)` values, instead of adding exceptions to the logic.

Reply:
Fine with me :)

Reply:
I saw that @travier [dropped](https://github.com/toolbx-images/images/commit/46eb11775cf608bfa764698efb888c37a780675d) the `ENV` from `github.com/toolbx-images/images`.  So, I took the liberty to do the same here.

I filed https://github.com/toolbx-images/images/pull/82 to rename the image on the registry to avoid user-visible breakage.